src/librustc/session/config.rs

```diff
@@ -879,20 +911,20 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                                 located",
                  "NAME=PATH"),
         opt::opt("", "sysroot", "Override the system root", "PATH"),
-        opt::multi("Z", "", "Set internal debugging options", "FLAG"),
-        opt::opt_u("", "error-format", "How errors and other messages are produced", "human|json"),
```

Comment:
Ah yeah I think that the "defaults" here are in reference to the method names. I like the idea about explicitly being stable, though, so I'll switch up the defaults so `foo` is truly unstable, `foo_s` is stable, and `foo_ubnr` is unstable-but-not-really